green-square-sequence/src/App.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react'
+import 'style.css'
+
+//set how many green squares you want displayed here
+let numOfSquares = 6
+
+export default function App() {
+  let [order, setOrder] = useState([])
+  let squares = [...Array(numOfSquares)]
+
+  const addElement = (num) => {
+    setOrder((order = order.concat(num)))
```

Comment:
The `order =` doesn't do anything here.  You just want to pass a new value into `setOrder`.

Also: What happens if this gets called twice with the same number? You need to protect against that (and remember the user could click twice _before_ the component gets a chance to rerender).